packages/io/amf-deserializer/index.js

```diff
@@ -35,7 +36,7 @@ const instantiate = require('./deserialize')
 const deserialize = (input, filename, options) => {
   const defaults = {
     output: 'jscad',
-    version: '0.0.0',
+    version: version,
```

Comment:
in modern js, you do not need to repeat the keys, so this should be 
```
{ output: 'jscad',
 version,
addMetaData: true
  }
```

Reply:
Understood. Let's catch these via lint. There are plenty.